src/utils/devhub-api-stitch.js

```diff
@@ -50,3 +50,13 @@ export const submitAcademiaForm = async academiaData => {
     );
     return result;
 };
+
+export const submitDevhubProject = async (metadata, object, segmentData) => {
+    const result = await callDevhubAPIStitchFunction(
```

Comment:
Is there a reason for storing the result in a variable here? Just curious

Reply:
No reason other than to match the existing structure of the other functions in this file. Previously it just returned the promise directly